### DISCUSSION TITLE
What is a good way to access the file_packager script?

### DISCUSSION THREAD

**Discussion by jspanchu** [emscripten-core/emscripten#21770] [Q&A]
In a python script, I'd like to run `file_packager.py` to preload files myself. This is what I do right now:

```python
EM_FILE_PACKAGER = Path(os.environ["EMSDK"]).joinpath("upstream/emscripten/tools/file_packager.py")
subprocess.call([sys.executable, EM_FILE_PACKAGER, ..])
```

Is there a better env var? I expect going down "upstream" wouldn't work in a developer setup.

**Comment by sbc100** [comment]
I would look for `emcc` in your path and then find `file_packager.py` relative to it.  

I guess maybe should create `emcc_file_packager` in top level dir so that you can just find that in your path directly?

**Reply by jspanchu** [reply]
> I guess maybe should create emcc_file_packager in top level dir so that you can just find that in your path directly?

That would be very convenient!

**Comment by sbc100** [comment]
BTW to find emcc in the `PATH` in python you can use `shutil.which('emcc')`
